great_expectations/data_context/data_context.py

```diff
@@ -282,6 +283,15 @@ def __init__(self, project_config, context_root_dir=None, runtime_environment=No
     def _build_store(self, store_name, store_config):
         module_name = "great_expectations.data_context.store"
         try:
+            # Set expectations_store.store_backend_id to given data_context_id if it doesnt exist
```

Comment:
Could we add a slightly more detailed explanation? The if statement conditions don't seem to match the comment - I understand that they are the same thing, but it could be confusing.

Reply:
I made an edit in https://github.com/great-expectations/great_expectations/pull/2046/commits/868a0b3183ccc14b4663383835782d2896281ca8 - let me know if it makes sense or happy to edit / add!